modules/text_server_adv/SCsub

```diff
@@ -508,6 +515,7 @@ env_text_server_adv.Append(
     CPPPATH=[
         "#thirdparty/harfbuzz/src",
         "#thirdparty/icu4c/common/",
+        "#thirdparty/icu4c/i18n/",
```

Comment:
Should be listed in `thirdparty/README.md` too.

Reply:
Individual file from the `i18n` are listed in the README.md (it's not the whole folder, just a tiny part), or there's some other place  it should be added to I'm missing?

Reply:
No it's all good now, but the files weren't listed yet when I commented (or I missed it).